pandas/core/sorting.py

```diff
@@ -517,16 +518,22 @@ def get_flattened_list(
     return [tuple(array) for array in arrays.values()]
 
 
-def get_indexer_dict(label_list, keys):
+def get_indexer_dict(
+    label_list: List[ABCIndex], keys: List[np.ndarray]
```

Comment:
Typing using the ABC classes is no different than `Any`; you can see this using mypy's reveal_type(label_list). In any case, I think label_list is also a list of ndarrays.

Reply:
Thanks for the info with ABC classes. I confused keys and label_list, it is the other way round. Thanks very much.